lib/workers/repository/update/branch/get-updated.ts

```diff
@@ -37,6 +37,7 @@ export async function getUpdatedPackageFiles(
     const packageFile = upgrade.packageFile!;
     const depName = upgrade.depName!;
     const newVersion = upgrade.newVersion!;
+    const currentVersion = upgrade.currentVersion!;
```

Comment:
please add a todo here, as I think this can be undefined at runtime ( docker image without tag and digest update /pin)